### DISCUSSION TITLE
List of features to do

### DISCUSSION THREAD

**Discussion by Olyno** [Olyno/supabase-community-bot#2] [General]
The community bot project is just starting, and we need to choose which features to add.

I've seen some proposals on the discord, such as:
 - Detecting messages in #chit-chat and sending an automatic message to redirect members to the help room
 - Add a "/done" command that would update the post title in #help by adding the prefix "[resolve]" in front of it, and closing the post.
 - Add a "/doc" command to send links from the doc.
 - Add a reputation system.

That said, these are just ideas, but we would like to hear from the whole community.

Each idea below will be added a thread below. The most up-voted threads will be the priorities. Please feel free to give feedback on the different ideas, especially if you don't want one of the ideas. You can also add a new idea for a feature you want to see added to the bot.

**Comment by Olyno** [comment]
## Automatic help message

Detecting messages in #chit-chat and sending an automatic message to redirect members to the help room

**Reply by NanoCode012** [reply]
In case of false positives / negatives (hard to detect messages), I would like to propose a public accessible command (`/help` or `/help <name>`) that would trigger the bot to return a standard response to redirect to help.

**Reply by Olyno** [reply]
Good point, it seems good to me to add an extra command

**Comment by Olyno** [comment]
## A "/done" command

Add a "/done" command that would update the post title in #help by adding the prefix "[resolve]" in front of it, and closing the post.

**Comment by Olyno** [comment]
## A "/doc" command

Add a "/doc" command to send links from the doc depending on terms we use. For example: "/doc auth".

**Comment by Olyno** [comment]
## A reputation system

Add a reputation system. Each people contributing to the Discord server get points depending on the activity. We could imagine some rewards if it's implemented.
